Xenakios/TakeRenaming.cpp

```diff
@@ -27,26 +27,27 @@
 
 #include "stdafx.h"
 
+#include "../Breeder/BR_Util.h" // GetSourceType()
 #include "../SnM/SnM_Dlg.h"
 #include "../SnM/SnM_Util.h" // SNM_DeletePeakFile()
 
 #include <WDL/localize/localize.h>
 
 using namespace std;
 
-typedef struct 
+struct t_renameparams
 {
 	int mode; // 0 for rename take, 1 for rename media source, 2 for both
 	string OldName;
 	string NewName;
 	int DialogRC;
-	int takesToRename;
+	size_t takesToRename;
 	int curTakeInx;
 	bool batchnaming;
 
-} t_renameparams;
+} ;
 
-t_renameparams g_renameparams;
+t_renameparams g_renameparams = {};
```

Comment:
Hm..if I remove the `= {}` I get these warnings. Why?
![sws_-_Microsoft_Visual_Studio_2021-01-26_16-51-53](https://user-images.githubusercontent.com/7658194/105869063-222f0600-5ff7-11eb-8068-76f4874e4aac.jpg)

Reply:
That comes from Visual Studio's static code analyzer rather than the compiler (Build + IntelliSense). No idea why it's triggered even though `g_renameparams` is global...